signalfx/util.go

```diff
@@ -208,7 +208,7 @@ func getColorScaleOptions(d *schema.ResourceData) []*chart.SecondaryVisualizatio
 }
 
 func getValueUsingMaxFloatAsDefault(v float64) *float32 {
-	if v >= math.MaxFloat32 || v <= -math.MaxFloat32 {
```

Comment:
The function name seems misleading, it doesn't "use max float as default" anywhere in this function.

Reply:
Changed!